en-US/N.xml

```diff
@@ -6,6 +6,15 @@
 <chapter id="n">
 	<title>N</title>
 	 <variablelist>
+		 <varlistentry id="navigate-to">
+ 			<term>navigate to</term>
+ 			 <listitem>
+ 				<para>
+					Use "Navigate to" when moving through multiple GUI options because it covers all cases where you might have to click, point to, select, or otherwise make a series of selections to initiate an action. For example, "From the OpenShift web console, navigate to Monitoring → Alerting."
```

Comment:
I would add a second para here that says something like "Do not use 'Go to' or 'point to' or other variations."

Reply:
Good advice David, that's done.